varnish/Dockerfile

```diff
@@ -6,5 +6,4 @@ RUN /tmp/docker-install.sh \
  && rm -rvf /tmp/*
 
 EXPOSE 6081
-ENTRYPOINT ["/usr/local/bin/chaperone"]
-CMD ["--user", "varnish"]
+CMD ["/usr/local/bin/chaperone", "--user", "varnish"]
```

Comment:
Yes, the entrypoint is always executed even if you want to start the container with a different command (for example "sh" or "bash"). Moving chaperone to the CMD allows you to more easily start the container with a totally different command (for whatever reason).